modules/catalog/src/test/java/org/apache/ignite/internal/catalog/CatalogManagerSelfTest.java

```diff
@@ -217,6 +217,22 @@ public void testEmptyCatalog() {
         assertThat(manager.latestCatalogVersion(), is(0));
     }
 
+    @Test
+    public void assignsSuccessiveCatalogVersions() {
+        CompletableFuture<Integer> version1Future = manager.execute(simpleTable(TABLE_NAME));
+        assertThat(version1Future, willCompleteSuccessfully());
```

Comment:
Can you do `willBe(1)` and etc?

Reply:
There is a test that checks explicitly that `execute()` returns successive version numbers. Non of other tests actually needs to know the completion value, it's enough to verify that the future completes with any value. If I add that, this will make the test very rigid for no reason.